app/helpers/arclight_helper.rb

```diff
@@ -225,4 +231,15 @@ def generic_should_render_field?(config_field, document, field)
   def generic_render_document_field_label(config_field, document, field: field_name)
     send(:"render_document_#{config_field}_label", document, field: field)
   end
+
+  private
+
+  def build_repository_link(document)
+    repository_path = document.repository_config&.slug
+    if repository_path.present?
```

Comment:
Do we need to check this? My understanding was that `slug` is a required property of `Arclight::Repository`

Reply:
in the fixtures, the large collection sample doesn't have a slug so this did not work without the check.

Reply:
Oh weird, ok!